### PR TITLE
RequestSyncDeliveryCache service stub

### PR DESCRIPTION
**Pull request by QuackingCanary**
RequestSyncDeliveryCache service stub to get Pikmin 4 booting

### CODE REVIEW

File: app/src/main/cpp/skyline/services/bcat/IDeliveryCacheProgressService.cpp [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2023 Strato Team and Contributors (https://github.com/skyline-emu/)
```
Comment (PabloG02) [inline app/src/main/cpp/skyline/services/bcat/IDeliveryCacheProgressService.cpp:2]:
```suggestion
// Copyright © 2023 Strato Team and Contributors (https://github.com/strato-emu/)
```

File: app/src/main/cpp/skyline/services/bcat/IDeliveryCacheProgressService.cpp [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        Logger::Debug("System Event Handle: 0x{:X}", handle);
+
+        response.copyHandles.push_back(handle);
+        return {};
+    }
+
+    Result IDeliveryCacheProgressService::GetImpl(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response) {
+        return {};
+    }
+}
```
Comment (lynxnb) [inline app/src/main/cpp/skyline/services/bcat/IDeliveryCacheProgressService.cpp:21]: New line at the end of a file.

File: app/src/main/cpp/skyline/services/bcat/IDeliveryCacheProgressService.h [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        Result GetEvent (type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response);
+
+        Result GetImpl (type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response);
+
+        SERVICE_DECL(
+        SFUNC(0x0, IDeliveryCacheProgressService, GetEvent),
+        SFUNC(0x1, IDeliveryCacheProgressService, GetImpl),
+        )
+    };
+}
```
Comment (lynxnb) [inline app/src/main/cpp/skyline/services/bcat/IDeliveryCacheProgressService.h:25]: New line at the end of each file.

**Review by lynxnb — CHANGES_REQUESTED** [review]
Thanks! I'll merge after you squash into 1 commit.

**Review by lynxnb — APPROVED** [review]

**Pull request merged**